modyn/tests/storage/internal/grpc/storage_service_impl_test.cpp

```diff
@@ -459,19 +459,16 @@ TEST_F(StorageServiceImplTest, TestGetNumberOfSamplesInDatasetWithRange) {
   ASSERT_NO_THROW(result = StorageServiceImpl::get_number_of_samples_in_dataset_with_range(1, session));
   ASSERT_EQ(result, 2);
 
-  ASSERT_NO_THROW(result = StorageServiceImpl::get_number_of_samples_in_dataset_with_range(1, session, 0, 0));
```

Comment:
Do we have a test now for having the default vs having explicit 0? If not, could you extend this test to test not set vs 0 as well?

Reply:
sure!